engines/pebble/pebble.go

```diff
@@ -3,19 +3,76 @@ package pebble
 import (
 	"errors"
 
+	"github.com/cockroachdb/pebble"
 	"github.com/rotationalio/honu/config"
+	engine "github.com/rotationalio/honu/engines"
+	"github.com/rotationalio/honu/iterator"
+	opts "github.com/rotationalio/honu/options"
 )
 
-func Open(conf config.ReplicaConfig) (*PebbleEngine, error) {
-	return &PebbleEngine{}, errors.New("not implemented yet")
+type PebbleEngine struct {
+	pebble *pebble.DB
 }
 
-type PebbleEngine struct{}
+// TODO: Allow Passing Pebble Options
+func Open(path string, conf config.ReplicaConfig) (_ *PebbleEngine, err error) {
+	engine := &PebbleEngine{}
+	if engine.pebble, err = pebble.Open(path, nil); err != nil {
+		return nil, err
+	}
+	return engine, nil
+}
 
+//Returns a string giving the engine type.
 func (db *PebbleEngine) Engine() string {
 	return "pebble"
 }
 
+//Close the database.
 func (db *PebbleEngine) Close() error {
-	return errors.New("not implemented yet")
+	return db.pebble.Close()
+}
+
+// Get the latest version of the object stored by the key.
+func (db *PebbleEngine) Get(key []byte, options ...opts.SetOptions) (value []byte, err error) {
+	if len(options) > 0 {
+		return nil, errors.New("pebble does not take read options")
+	}
```

Comment:
We may have options besides engine specific options; e.g. options for returning metadata or modifying indices. However, those will likely be captured at the Honu level rather than the engine level, but it's probably best to simply ignore the options here since they'll cause no harm. 

```suggestion
```